src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -408,6 +408,13 @@ private function collectPaths(ApiResource $resource, ResourceMetadataCollection
                 }
             }
 
+            if (true === $overrideResponses && !isset($existingResponses[403]) && $security = $operation->getSecurity()) {
+                $openapiOperation = $openapiOperation->withResponse(403, new Response(\sprintf(
+                    'Forbidden: You must meet the following condition(s) to access this resource: %s.',
+                    $security
+                )));
```

Comment:
```suggestion
                $openapiOperation = $openapiOperation->withResponse(403, new Response('Forbidden'));
```

I don't think that we should disclose any user configuration inside the documentation, WDYT?

Reply:
@soyuka Solved. All that remains is to add tests.